examples/browser/src/index.ts

```diff
@@ -0,0 +1,56 @@
+import { Mesh, OrderEvent, SignedOrder, BigNumber } from '@0x/mesh-browser';
+
+(async () => {
+    // Confingure Mesh to use mainnet and Infura.
+    const mesh = new Mesh({
+        ethereumRPCURL: 'https://mainnet.infura.io/v3/af2e590be00f463fbfd0b546784065ad',
```

Comment:
You sure you want to leave this in here?

Reply:
🤷‍♂It's probably fine. I'm not using that key for anything at the moment. It has the advantage of making the example easily runnable.